src/inference/include/openvino/runtime/properties.hpp

```diff
@@ -607,6 +608,13 @@ static constexpr Property<element::Type, PropertyMutability::RW> kv_cache_precis
  */
 static constexpr Property<float, PropertyMutability::RW> activations_scale_factor{"ACTIVATIONS_SCALE_FACTOR"};
 
+/** @brief  Hint for device to use model compiled blob.
+ * @ingroup ov_runtime_cpp_prop_api
+ *
+ * The property is used pass compiled blob as ov::Tensor.
+ * The blob can be regular or weightless model. The `weights_path` property is hint where to look for weights.
+ */
+inline constexpr Property<Tensor, PropertyMutability::RW> compiled_blob{"COMPILED_BLOB"};
```

Comment:
BTW, I don't see a logic that in case of `hint::compiled_blob` is passed, then Core tries to call `import_model` first:
- is `hint::compiled_blob` passed?
  - try `plugin.import_model(hint::compiled_blob, weights_path / hint::model_ptr via properties to restore weights if required)`
  - in case of success => return CompiledModel
  - in case of failure => go to pugin.compile_model
- hint::compiled_blob is not passed => go to current flow.

Interesting case, when both `cache_dir` and `hint::compile_blob` is used - which one is preferable? I suppose `hint::compile_blob` still has higher priority.

Reply:
Added logic to give priority for `hint::compiled_blob`